### DISCUSSION TITLE
Using image sequences in tsai

### DISCUSSION THREAD

**Discussion by Samcro5C** [timeseriesAI/tsai#513] [Q&A] [answered]
Hello, I was just wondering if there is a possibility to use image sequences in tsai. As far as I understood, the input needs to be [# samples x # variables x sequence length]. For image sequences however, it would be something like [# samples x # channels x height x width x sequence length]. Is it in general possible to use an input like that? Furthermore, would it be possible to combine image sequences and floating point time series using something like the MultiInputNet?

Thanks in advance for the feedback!

**Comment by oguiza** [comment, marked as answer]
No, unfortunately, you can't use image sequences with tsai.
